website/docs/concepts/provider_lifecycles.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Provider Lifecycles
+---
+
+## When does my Provider get created and disposed?
+
+The states that all different types of providers can go through are the same:
+
+- Disposed / Uninitialized
```

Comment:
There are more life cycles related to running/paused (cf onCancel/onResume)

Reply:
Added that below, you should probably verify that I understand these particular lifecycle hooks appropriately.